caikit/config/config.yml

```
@@ -195,6 +195,12 @@ runtime:
         output_dir: training_output
         save_with_id: true
 
+    # Version details to retrieve
+    version_info:
+        python_packages:
+            all_packages: false
```

I'm the worst, but...

```suggestion
            all: false
```